common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Work.scala

```diff
@@ -189,14 +189,14 @@ object WorkState {
     // work to be performed by the pipeline
     sourceModifiedTime: Instant,
     mergeCandidates: List[MergeCandidate[IdState.Identifiable]] = Nil,
-    internalWorkStubs: List[InternalWork.Source] = Nil
+    internalWorkStubs: List[InternalWork.Source] = Nil,
+    relations: Relations = Relations.none
```

Comment:
How much do you understand the different states here (Source, Identified, Merged, etc.)?

This is definitely the right place to put this value, but I think it's going to be dropped at the next stage of the pipeline – the Identified/Merged states always assume that `Relations` is going to be `none` (and the merger/relation embedder may ignore it too, although that's a bit more complicated).

I think you need to extend the Identified and Merged states additionally, then update the WorkFsm described further down in this file.

(This is another case of where the internal model is quite brittle and unwieldy, but it's what we have for now.)

Reply:
Yes. I think I forgot that this is used to communicate between applications by DB & queue, rather than by in-memory copies in a single application.

Reply:
There is something I find a bit odd here - Most of the states have some `override val` lines, which seem unnecessary, as the val in the trait is only declared and not defined.  Is there some magic I'm missing?

Reply:
I think the compiler complains if the values aren't defined in the case classes? But I don't recall there being any special magic here; it's possible they're unnecessary. Try removing them and let me know!